tcms/testplans/static/testplans/js/get.js

```diff
@@ -358,6 +358,17 @@ function toolbarEvents() {
         }
 
     });
+
+    $('#bulk-clone-button').click(function() {
+        let selectedCases = getSelectedTestCases();
+
+        if (!selectedCases.length) {
+            alert($('#test_plan_pk').data('trans-no-testcases-selected'));
+            return;
+        }
+
+         window.location.replace(`/cases/clone?case=${selectedCases.join('&case=')}`);
```

Comment:
Use `.assign` instead of replace b/c it messes up the back button which is what #1054 is about.